pkg/virt-handler/vm.go

```diff
@@ -1466,8 +1462,7 @@ func (c *VirtualMachineController) recordPhaseChangeEvent(vmi *v1.VirtualMachine
 
 func calculatePausedCondition(vmi *v1.VirtualMachineInstance, reason api.StateChangeReason) {
 	now := metav1.NewTime(time.Now())
-	switch reason {
-	case api.ReasonPausedMigration:
+	if isVMIPausedDuringMigration(vmi) {
```

Comment:
Personally I'd leave the switch in place and just have this as a standalone conditional with an early return to avoid the `} else if {` mess below.